tests/test-recipes/metadata/jinja2/meta.yaml

```diff
@@ -0,0 +1,14 @@
+package:
+  name: conda-build-test-jinja2-in-recipe
+  version: 1.0
+
+build:
+  number: 0
+
+requirements:
+    build:
+        - jinja2
```

Comment:
You should add python here so that it builds against the same Python version as conda build.